content/general/concepts/cryptocurrency/cryptocurrency.md

```diff
@@ -0,0 +1,89 @@
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Blockchain'
+  - 'Cryptocurrency'
+CatalogContent:
+  - 'introduction-to-blockchain'
+  - 'paths/computer-science'
+---
+
+A cryptocurrency is a digital asset or token secured by cryptography. Such assets are transferred on decentralized networks based on blockchain technology.
+
+Cryptocurrencies are not issued or controlled by any central authority, such as the federal bank of a sovereign nation as is the case with fiat money such as the US dollar. Cryptocurrencies are therefore protected from government interference, although this does not mean they cannot be manipulated by other parties.
+
+### Blockchain
+
+A blockchain is essentially a set of interconnected “blocks” that form an online ledger that is distributed across computers around the world. These linked blocks contain recorded transactions that are verified by nodes in the network. Nodes are the network computers running the software client that locally stores the blockchain data.
+
+The data is updated as transactions are stored in each block and every copy of the blockchain is updated simultaneously, keeping all the records identical and accurate. As the network relies on consensus mechanisms to verify transactions, it is almost impossible to forge a fraudulent transaction.
+
+### Mining
+
+Units of cryptocurrencies are created through a process known as mining, which involves using computing power to solve complex mathematical problems. The computer in the network that solves the problem the fastest generates the next block in the chain and is rewarded with units of that network, such as bitcoin. In the early days of Bitcoin, individual users were able to participate in mining, although nowadays the computing power necessary is only achieved through a powerful network of computers known as a “mining pool”.
+
+### Types
+
+Bitcoin is the most popular and widely known cryptocurrency, as well as the most valuable as measured by total market capitalization. It was created by an individual (or perhaps a group) called Satoshi Nakamoto and introduced via a white paper in 2008. With the success of Bitcoin, thousands of other cryptocurrencies have since been created with different attributes for various functions.
+
+Ethereum, the second most valuable cryptocurrency, features a smart contract platform that allows for numerous protocols including borrowing and lending services. There are also “stable coin” cryptocurrencies, which attempt to offer price stability and are backed by a reserve asset such as US dollars.
+
+There are also “altcoins”, some of which are forks of Bitcoin and others are new currencies, tokens, or assets that have been built from scratch. These altcoins include meme coins inspired by memes and internet jokes. Dogecoin, for example, was inspired by the Doge meme created from a viral photo of a Shiba Inu.
```

Comment:
```suggestion
There are also **altcoins**,some of which are forks of Bitcoin and others are new currencies, tokens, or assets that have been built from scratch. These altcoins include meme coins inspired by memes and internet jokes. Dogecoin, for example, was inspired by the "Doge meme" created from a viral photo of a Shiba Inu.
```

I think we include inline meme names using double quotes

Reply:
@KTom101 @Dusch4593 do we know if it's okay to bold words we want to bring attention to 💭

Reply:
Great, thanks for the feedback @yangc95. I'll wait until this is clarified then do all the changes at once.

Reply:
@yangc95 @0xTomas Per our [Markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/#bold), bold should be used for new terms, keywords, or concepts (but not for general emphasis).

Reply:
Thanks @Dusch4593, I have updated these changes.